packages/gallery/package.py

```diff
@@ -75,3 +75,11 @@ def setup_dependent_build_environment(self, env, dependent_spec):
         env.prepend_path("CET_PLUGIN_PATH", prefix.lib)
         # Cleaup.
         sanitize_environments(env, "CET_PLUGIN_PATH")
+
+    def setup_run_environment(self, run_env):
+        run_env.prepend_path("PATH", self.prefix.bin)
+        run_env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
+        run_env.append_path("FHICL_FILE_PATH", "{0}/fcl".format(self.prefix))
+        run_env.append_path("FW_SEARCH_PATH", "{0}/fw".format(self.prefix))
+        run_env.append_path("CET_PLUGIN_PATH", self.prefix.lib)
```

Comment:
`gallery` is just a library--it provides no executables, FHiCL files or plugins.  I think the only adjustment to include is the one for `ROOT_INCLUDE_PATH`.

Reply:
Done